web/src/PredictImpactMode.svelte

```diff
@@ -46,8 +75,16 @@
     <p>
       This mode estimates the impact of all your changes on traffic around the
       entire area. It's based on many assumptions and must be interpreted very
-      carefully.
+      carefully. <Link on:click={() => ($mode = { mode: "debug-demand" })}>
+        Explore the origin/destination demand data used
+      </Link>
     </p>
+
+    <label>
+      <input type="checkbox" bind:checked={$fastSample} />
```

Comment:
Not a big deal, but a radio might be better at communicating this as a tradeoff:

- [x] Calculate quickly
- [ ] Calculate more accurately

Reply:
Good point, done